apps/spikeConverter.cpp

```diff
@@ -93,6 +93,13 @@ int main( int argc, char* argv[] )
                   << brion::Version::getString() << std::endl;
         return EXIT_SUCCESS;
     }
+    if( vm["input"].as< std::string >() == vm["output"].as< std::string >( ))
+    {
+        std::cerr << "Cowardly refusing to convert "
+                  << vm["input"].as< std::string >() << " onto itself"
+                  << std::endl;
+        return EXIT_FAILURE;
+    }
```

Comment:
Since you're touching this code, I'd remove all default values for input and output. These values are completely useless in my opinon, furthermore, after I rescued the positional arguments it took me some time to realize that the converter was not working as before and that positional arguments were ignored (good that I used the tool from your PR for a test and I realized this change in behaviour, because it'd have been bad to merge it). If the tool is run without sufficient parameters, it should print an error and help, not do some default non-informative useless action.